src/commands/commerce/store/create.ts

```diff
@@ -256,11 +256,12 @@ export class StoreCreate extends SfdxCommand {
         if (
             !this.varargs['buyerEmail'] ||
             (this.varargs['buyerEmail'] as string).indexOf('scratchOrgBuyerUsername.replace') >= 0
-        )
-            this.varargs['buyerEmail'] = `${os.userInfo().username}+${(this.flags['buyer-username'] as string).replace(
-                '@',
-                'AT'
-            )}@salesforce.com`;
+        ) {
+            let buyerEmail = os.userInfo().username + (this.flags['buyer-username'] as string);
```

Comment:
```
`${os.userInfo().username}+${(this.flags['buyer-username'] as string).replace(
                '@',
                'AT'
            )}@salesforce.com`
```

We lose the '+' sign between `os.userInfo().username` and `this.flags['buyer-username']` since your using string concatenation instead of string templating, is that breaking?